ImageFilterEditor/ImageFilterEditor/ViewController.swift

```diff
@@ -0,0 +1,368 @@
+//
+//  ViewController.swift
+//  ImageFilterEditor
+//
+//  Created by Karen Rodriguez on 5/4/20.
+//  Copyright © 2020 Hector Ledesma. All rights reserved.
+//
+
```

Comment:
This file is huge! Taking my advice to heart about just getting something done and refactoring later... good! 😅